livekit-server/templates/ingress.yaml

```diff
@@ -30,11 +25,22 @@ metadata:
     kubernetes.io/ingress.class: nginx
     cert-manager.io/cluster-issuer: {{ .Values.loadBalancer.clusterIssuer }}
   {{- end }}
-{{- with .Values.loadBalancer }}
+{{- if semverCompare ">=1.19-0" .Capabilities.KubeVersion.GitVersion }}
```

Comment:
I had to put this in a weird place in the file because the templating engine didn't parse correctly when the apiVersion and relevant specs were in separate conditionals. I choose this way because it did not require the replication of the metadata or the TLS code at the expense of putting the apiVersion in a bit of a strange place in the file.